README Updates (#4)
TonicAI/textual: README.md

</h1>

<p align="center">Unblock AI initiatives by maximizing your free-text assets through realistic data de-identification and high quality data extraction 🚀</p>

<p align="center">
    <a href="https://www.python.org/">
      <img alt="Build" src="https://img.shields.io/badge/Made%20with-Python-1f425f.svg?color=purple">
    </a>
    <a href="https://github.com/tonicai/textual_sdk_internal/blob/master/LICENSE">
      <img alt="License" src="https://img.shields.io/badge/license-MIT-blue">
    </a>
    <a href='https://tonic-ai-textual-sdk.readthedocs-hosted.com/en/latest/?badge=latest'>
      <img src='https://readthedocs.com/projects/tonic-ai-textual-sdk/badge/?version=latest' alt='Documentation Status' />
    </a>
</p>

<p align="center">
  <a href="https://tonic-ai-textual-sdk.readthedocs-hosted.com/en/latest/">Documentation</a>
  |
  <a href="https://textual.tonic.ai/signup">Get an API key</a>
  |
  <a href="https://github.com/tonicai/textual_sdk/issues/new?labels=bug&template=bug-report---.md">Report a bug</a>
  |
  <a href="https://github.com/tonicai/textual_sdk/issues/new?labels=enhancement&template=feature-request---.md">Request a feature</a>
</p>

> Textual makes it easy to build safe AI models and applications on sensitive customer data.  It is used across industries with a primary focus on finance, healthcare, and customer support.  Build safe models by using Textual to identify customer PII/PHI then generate synthetic text and documents which you can use to train your models without inadvertently embedding PII/PHI into your model weights.

Reviewer: ```suggestion
Textual makes it easy to build safe AI models and applications on sensitive customer data. It is used across industries, with a primary focus on finance, healthcare, and customer support. Build safe models by using Textual to identify customer PII/PHI, then generate synthetic text and documents that you can use to train your models without inadvertently embedding PII/PHI into your model weights.
```

> Textual comes built-in with data pipeline functionality so it scales with you.  Use our SDK to redact text or to extract relevant information form complex documents prior to building your data pipelines.

Reviewer: ```suggestion
Textual comes with a built-in data pipeline functionality so that it scales with you. Use our SDK to redact text or to extract relevant information from complex documents before you build your data pipelines.
```

## Key Features

> - 🔎 NER. Our models are fast and accurate.  Use them on real-world, complex and messy unstructured data to find just the entities you care about.

Reviewer: ```suggestion
- 🔎 NER. Our models are fast and accurate. Use them on real-world, complex, and messy unstructured data to find the exact entities that you care about.
```

> - 🧬 Synthesis. We dont just find sensitive data, we synthesize it giving you a new version of your data most suitable for model training and AI development.

Reviewer: ```suggestion
- 🧬 Synthesis. We don't just find sensitive data. We also synthesize it, to provide you with a new version of your data that 
is suitable for model training and AI development.
```

> - ⛏️ Extraction. We support a variety of file formats, not just txt.  We can extract interesting data from PDF, DOCX, images, and more.

Reviewer: ```suggestion
- ⛏️ Extraction. We support a variety of file formats in addition to txt. We can extract interesting data from PDFs, DOCX files, images, and more.
```

In addition to processing files from your local system, you can reference files directly from Amazon S3. The ```parse_s3_file``` function call behaves the same as ```parse_file```, but requires a bucket and key argument to specify your specific file in Amazon S3. It uses boto3 to retrieve the files from Amazon S3.

_For more examples, refer to the [Textual SDK documentation](https://textual.tonic.ai/docs/index.html)_

<p align="right">(<a href="#readme-top">back to top</a>)</p>

## UI automation

The Textual UI supports file redaction and parsing. It provides an experience for users to orchestrate jobs and process files at scale. It supports integrations with various bucket solutions such as Amazon S3, as well as systems such as Sharepoint and Databricks Unity Catalog volumes.

> You can use the SDK for actions such as building smart pipelines (for parsing) and Dataset collections (for file redaction).

Reviewer: ```suggestion
You can use the SDK for actions such as building smart pipelines (for parsing) and dataset collections (for file redaction).
```
